repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/tracing/rewritetransid/system-model.cfg.xml

```diff
@@ -13,4 +13,5 @@
                       default="true"/>
         </destinations>
     </repose-cluster>
+    <tracing-header rewrite-header="true"/>
```

Comment:
I initially had it require the "enabled" attribute to make it clearer but took it out at the last minute.  If I made that required, then the options would be:

`--nothing--` : add tracing header, no rewrite
`<tracing-header enabled="true" />` : add tracing header, no rewrite
`<tracing-header enabled="false" />` : don't add tracing header
`<tracing-header enabled="true" rewrite-header="true" />` : add tracing header, rewrite if needed

Which would be a bit better than being in the situation where "`<tracing-header enabled="true" />`", "`<tracing-header />`" and nothing mean the same thing.

The other option is we also just make tracing-header a required element.  Explicit is better, right?  I'm not sure how I feel about making that a necessary change when updating to Repose 8, but I suppose this would be the time to do it.

I think putting a verb in the element name would be inconsistent.